composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/EmptyScreenContent.kt

```diff
@@ -5,9 +5,11 @@ import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import com.jetbrains.kmpapp.MR
-import dev.icerock.moko.resources.compose.stringResource
+import kmp_app_template.composeapp.generated.resources.*
```

Comment:
Usually we don't allow * imports, is this because it's generated?

Reply:
See also below

Reply:
Replaced it with individual imports.